tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py36,py27}-django{111, 20, 21},flake8
+envlist = {py36,py37,py38}-django{111, 22, 30},py27-django{111, 22},flake8
```

Comment:
Tests do take more time to run, but I don't see why not running them for Django 2.0 and 2.1 too. But this is not a blocker but a personal opinion.

Reply:
I choose not to run tests on these version, because they aren't officially supported anymore.
See [Django](https://www.djangoproject.com/download/)

I do understand that there are a multitude of projects out here using these versions for a reason, but AFAIK aren't there any breaking changes, between the versions, so one version should be enough.

Reply:
OK, agree.